README.md

```diff
@@ -1,3 +1,12 @@
+## Introduction
+
+**Welcome to Hub Automation!**
+
+This this collection of scripts and the repo are here for your self improvement and to allow for the sharing of automation amongst the IBM Quantum Network community of members. Please feel free to contribute your enhancements, and to propose new scripts, features, and fixes as appropriate.  While we (IBM Quantum will monitor this repo, and contribute as we are able, this is a community project. 
```

Comment:
I think there is a ')' missing after IBM Quantum.

Reply:
Updated. Should be good to go now.